src/Refactoring-Transformations/ReCompositeExtractMethodRefactoring.class.st

```diff
@@ -114,18 +114,19 @@ ReCompositeExtractMethodRefactoring >> applicabilityPreconditions [
 		  self preconditionTemporariesAreNotReadBeforeWritten.
 		  self preconditionOneAssignmentMaximum.
 		  self preconditionAssignmentsNotReadBeforeWritten.
-		  self preconditionSubtreeDoesNotContainsReturn }
+		  self preconditionSubtreeDoesNotContainsReturn.
+		  self preconditionHasSameExitPoint }
```

Comment:
I moved this precondition from breaking changes to applicability preconditions because it does not depend on the user input and can be computed before asking for the new method's name. Besides, I couldn't think of an effective choice associated to it